st2client/st2client/shell.py

```diff
@@ -75,6 +76,14 @@
     %(prog)s --debug run core.local cmd=date
 """.strip()
 
+NON_UTF8_LOCALE = """
+Locale %s with encoding %s which is not UTF-8 is used. This means that some functionality which
+relies on outputting unicode characters won't work.
+
```

Comment:
Just a minor note to not include line break here.
It looks inconsistent when more Warning messages are included.

See 
![st2-locale-warning](https://user-images.githubusercontent.com/1533818/39961541-1e66eca0-5630-11e8-8bf9-28610de6f0d5.png)

Reply:
Yep, agreed.